timeline-chart/src/components/time-graph-annotation.ts

```diff
@@ -0,0 +1,90 @@
+import * as PIXI from "pixi.js-legacy"
+
+import { TimeGraphComponent, TimeGraphElementPosition, TimeGraphComponentOptions } from "./time-graph-component";
+
+export interface TimeGraphAnnotationCoordinates extends TimeGraphComponentOptions {
```

Comment:
We might not need this, if annotations can be a TimeGraphRowElement or a sublass of it.

But then 'symbol' should be added to TimeGraphRowElementStyle.

Should row element be a parent of two classes, state and annotation? Or a single class that does both? What we decide should probably apply to both the *Component and the *Model, and perhaps *Style.